generate.sh

```diff
@@ -31,23 +32,16 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 
 platform=''
 version=''
-arch=''
 frameworks=()
 configuration='Debug'
+is_maccatalyst=false
+tool=false
 experimental=false
-
-output_dir="./GeneratedBindings"
+dotnet_version="net9.0"
```

Comment:
How do we decide which .NET version we're on?

Reply:
It is related to the version of the tooling:
```
local script_path="$scriptroot/artifacts/bin/Swift.Bindings/$configuration/$dotnet_version/Swift.Bindings.dll"
```